### ISSUE TITLE
Issue with Pluralizer

### ISSUE THREAD

**Issue by Foxandxss** [breeze-sandbox/northwind-core-ng#2]
I followed all the steps but I get a:

`Could not load file or assembly 'Bricelam.EntityFrameworkCore.Pluralizer, Culture=neutral, PublicKeyToken=null'. The system cannot find the file specified.`

After trying to scaffold the database. If I remove this assembly and I try again, it works (I lose the pluralization in the generation tho).

No clue what is wrong.

**Comment by steveschmitt** [comment]
The `Bricelam.EntityFrameworkCore.Pluralizer` package should install `Microsoft.EntityFrameworkCore.Design` as a dependency.  Is that happening?

**Comment by Foxandxss** [comment]
Yes, that is happening. For some reason the command tries to find the assembly but it doesn't find it. If I remove it, it just ignores it and work. Like an incomplete dependency or something.
